### DISCUSSION TITLE
home-manager: programs.git.delta.enable

### DISCUSSION THREAD

**Discussion by bbigras** [lovesegfault/nix-config#906] [Ideas]
https://github.com/lovesegfault/nix-config/blob/213f6a6fa74640259d335106b3e7b96018113117/users/bemeurer/core/git.nix#L9

Btw there's https://rycee.gitlab.io/home-manager/options.html#opt-programs.git.delta.enable

**Comment by lovesegfault** [comment]
Ah, good to know! Thanks!
